### ISSUE TITLE
Localize share image in promote tab

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/foundation.mozilla.org#3633]
@alanmoo to add more info

**Comment by alanmoo** [comment]
@TheoChevalier Is this for cases where there's text in the share image? My instinct is that we should instead have a general rule of "no text in images because all-the-reasons"

**Comment by TheoChevalier** [comment]
@alanmoo Yes, and I would agree… but there are cases where we just can’t avoid it.
Here’s a good recent example, we have localized graphics and sharing them on social media is the main CTA of the blog post https://foundation.mozilla.org/fr/campaigns/how-opt-out-human-review-your-voice-assistant-recordings/

Which resulted in this: https://twitter.com/staging_account/status/1179846731669880832

**Comment by alanmoo** [comment]
We should have a general rule of no text in images, and a strict rule of no text in OG images. That share could have been done without putting words into it. Embedding the text in the share image is terrible from an SEO standpoint anyway.
